src/javascript/app/pages/trade/markets.jsx

```diff
@@ -89,7 +89,7 @@ const submarket_order = {
     random_nightly : 21,
 };
 
-const submarketSort = (a, b) => {
+export const submarketSort = (a, b) => {
```

Comment:
we can put this in active symbols, so we have one source for active symbols related function?, and import it here

Reply:
@mustofa-binary I see what you mean, so you want the sorting to be part of the function `getSymbolsForMarket`, okay, so let me create a function for that